datamodel/changelogs/0001/05_data_model_extensions.sql

```diff
@@ -53,6 +53,17 @@ SET order_fct_hierarchic=
 		 ]
 	 ,code);
 
+-- this column is an extension to the VSA data model and facilitates filtering out primary features
+ALTER TABLE tww_vl.channel_function_hierarchic ADD COLUMN is_primary bool;
```

Comment:
@cymed With our naming convention it should be _is_primary - isn't it?

Reply:
or tww_is_primary (see https://github.com/teksi/wastewater/pull/164)

Reply:
@cymed then let's do it like in #164 - then we are consistent